### DISCUSSION TITLE
一点个人意见，作者别怂啊，放个毛泽东讲话并起不到保护维权人的作用

### DISCUSSION THREAD

**Discussion by hugetiny** [WeNeedHome/SummaryOfLoanSuspension#1056] [General]
这个真的没人在乎，不管是官方还是草民。
然后像我比较讨厌土共的人，github看到这个有点恶心

**Comment by clleo888** [comment]
这种心情是理解的, 但是邪党口裡也会说孔子大道理, 只是它们只说不做而已, 那道理本身都是没错的。
只要是有道理, 真心相信身体力行, 还是没问题的。
**起来！不愿做奴隶的人们！
中华民族到了最危险的时候，
每个人被迫着发出最后的吼声。**

**Comment by JayChou55** [comment]
毛迫害的人 可是以万作单位的。。。沙比没得救的
